avm/res/dev-ops-infrastructure/pool/main.bicep

```diff
@@ -293,7 +293,7 @@ type organizationProfileType = {
   @description('Optional. The type of permission which determines which accounts are admins on the Azure DevOps pool.')
   permissionProfile: {
     @description('Optional. Determines who has admin permissions to the Azure DevOps pool.')
```

Comment:
If I am not mistaken, the value is now mandatory. Therefore, Optional should be Required.

Reply:
Yes, correct. Thanks for checking @ReneHezser!

Reply:
Fixed